### PR TITLE
Our story video bg

### PR DESCRIPTION
**Pull request by tulupvica**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -157,7 +157,23 @@ <h3 class="main-title section-our-story__title">Our story</h3>
...
       </section>
+      <!--end section-our-story-->
+
+      <section class="story-video-bg">
+        <video poster="assets/images/bg-1-1600px.jpg" class="story-video-bg__video">
+          <source src="assets/video/video.mp4" type="video/mp4">
+        </video>
+        <div class="story-video-bg__buttons">
+          <button class="story-video-bg__btn-play" type="button">
+            <img src="assets/images/play.png" alt="play">
```
Comment (ymazhar) [inline index.html]: Not critical, but better use SVG

File: src/scss/components/story-video-bg.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.story-video-bg {
+
+  $bg-gradient: linear-gradient(to right, rgba(9,5,47,0.85) 0%,rgba(49,29,94,0.85) 100%);
```
Comment (ymazhar) [inline src/scss/components/story-video-bg.scss]: Make scope of variables in the start of the file

File: index.html (hunk 1 above)
Comment (tulupvica) [reply index.html]: I didn't get to export svg from the design

File: src/scss/components/story-video-bg.scss (hunk 2 above)
Comment (tulupvica) [reply src/scss/components/story-video-bg.scss]: fixed

File: index.html (hunk 1 above)
Comment (tulupvica) [reply index.html]: fixed

File: src/scss/components/story-video-bg.scss (hunk 2 above)
Comment (tulupvica) [reply src/scss/components/story-video-bg.scss]: fixed

File: index.html (hunk 1 above)
Comment (tulupvica) [reply index.html]: fixed

**Pull request merged**